erpnext/accounts/doctype/pricing_rule/utils.py

```diff
@@ -157,6 +157,15 @@ def get_other_conditions(conditions, values, args):
 	return conditions
 
 def filter_pricing_rules(args, pricing_rules, doc=None):
+	#coupon_code_based_pricing_rule
```

Comment:
this function is too big!

cc @rohitwaghchaure

Reply:
Refactoring of pricing rule functions can be resolve in separate PR